### PR TITLE
Fixing getting started

### PR DESCRIPTION
**Pull request by ArtursKadikis**

### CODE REVIEW

**Comment by ArtursKadikis** [comment]
@turtledreams check this and then push to zendesk

File: getting_started/current.md [hunk 1]
```diff
@@ -42,10 +41,10 @@
...
 <p>
   Most Countly SDKs send some user properties together with the session request&nbsp;
-  which is triggered automatically when the app is launched, user lands on a website,
-  or manually in some cases such as server to server SDK usage. These properties,
-  which we call default user properties, are metadata including but not limited
-  to app version, device, platform, platform version.&nbsp;
+  triggered automatically when the app is launched, the user lands on a website,
+  or manually in some cases such as server-to-server SDK usage. These properties,
+  which we call default user properties, are metadata, including but not limited
+  to the app, device, platform, and platform.&nbsp;
```
Comment (turtledreams) [inline getting_started/current.md:47]: why?

File: getting_started/current.md [hunk 2]
```diff
@@ -66,10 +65,10 @@
   <a href="/hc/en-us/articles/360037639931" target="_blank" rel="noopener">Formulas</a>.
 </p>
 <p>
-  Custom user properties consist of key value pairs, where value can be a string,
-  number, boolean, or an array. There are various modifiers available in the SDKs
-  such as set, set once, push unique, increment, multiply in order to update these
-  as you need. For example, you can store account levels of your users in an
+  Custom user properties consist of key-value pairs, where the value can be a string,
+  number, boolean, or array. There are various modifiers available in the SDKs,
+  Such as set, set once, push unique, increment, and multiply to update these
```
Comment (turtledreams) [inline getting_started/current.md:70]: such?

File: getting_started/current.md [hunk 3]
```diff
@@ -291,18 +288,18 @@
 <p>
   With mobile devices, there are 3 different modalities that a user can interact
   with: phone app, phone widget, and watch app. When designing a product that spans
-  all three worlds, the goal is usually to track the user's lifecycle across all
```
Comment (ArtursKadikis) [inline getting_started/current.md:294]: fishy

File: getting_started/current.md [hunk 4]
```diff
@@ -199,45 +197,44 @@
...
   devices. Therefore you need to use the same device ID on all of that user's devices.
-  To achieve that you would have an internal ID of your own system or a derivative
-  value of it that you can reliably generate.
+  To achieve that, you would have an internal ID of your system or a derivative
+  value that you can reliably generate.
 </p>
 <p>
   The first time you initialize a Countly SDK, you would allow the SDK to generate
-  the device ID. When the user logs in to your app, you would change the device
+  the device ID. When the user logs in to your app, you will change the device
```
Comment (ArtursKadikis) [inline getting_started/current.md:215]: wrong would->will

File: getting_started/current.md [hunk 5]
```diff
@@ -199,45 +197,44 @@
...
   ID to the new ID that you generate. If this is the first time the user logs in,
-  you would change the device ID with merging. On any other follow-up logins, you
-  would change the device ID without merging. This way you can assign the events
+  you will change the device ID with merging. You
```
Comment (ArtursKadikis) [inline getting_started/current.md:217]: wrong would->will

File: getting_started/current.md [hunk 6]
```diff
@@ -291,18 +288,18 @@
 <p>
   With mobile devices, there are 3 different modalities that a user can interact
   with: phone app, phone widget, and watch app. When designing a product that spans
-  all three worlds, the goal is usually to track the user's lifecycle across all
+  all three worlds, the goal is to track the user's lifecycle across all
```
Comment (turtledreams) [inline getting_started/current.md:291]: sometimes not?

File: getting_started/current.md (hunk 2 above)
Comment (ArtursKadikis) [inline getting_started/current.md:70]: why "such" capitalized?

File: getting_started/current.md [hunk 7]
```diff
@@ -42,10 +41,10 @@
 </p>
 <p>
   Most Countly SDKs send some user properties together with the session request&nbsp;
-  which is triggered automatically when the app is launched, user lands on a website,
```
Comment (ArtursKadikis) [inline getting_started/current.md:45]: Needs the "which is"

File: getting_started/current.md (hunk 1 above)
Comment (ArtursKadikis) [inline getting_started/current.md:47]: missing version

File: getting_started/current.md [hunk 8]
```diff
@@ -4,20 +4,19 @@
...
 <h1>User/device identification</h1>
 <p>
-  By default, Countly generates an anonymous identifier to uniquely identify a
-  device or browser. Total users metric for any period is based on unique number
-  of device ids Countly Server receives requests from. Similarly, users list in
+  By default, Countly generates an anonymous identifier to uniquely identify a device or browser. The total users metric for any period is based on a unique number
+  of device ids the Countly Server receives requests from. Similarly, users list in
   <a href="/hc/en-us/articles/360037630571" target="_blank" rel="noopener">User Profiles</a>
-  section is a list of anonymous devices Countly Server is receiving data from.
+  The section lists anonymous devices from which the Countly Server is receiving data.
```
Comment (ArtursKadikis) [inline getting_started/current.md:19]: wrong "The"

File: getting_started/current.md (hunk 2 above)
Comment (ArtursKadikis) [reply getting_started/current.md:70]: we should put the operators in quotes

File: getting_started/current.md (hunk 4 above)
Comment (ArtursKadikis) [reply getting_started/current.md:215]: "we recommend"

File: getting_started/current.md (hunk 6 above)
Comment (ArtursKadikis) [reply getting_started/current.md:291]: normally or ideally

**Review by turtledreams — APPROVED** [review]

**Pull request merged**